### ISSUE TITLE
Feature request: blank theme / CSS "themable"

### ISSUE THREAD

**Issue by dagnelies** [jdorn/json-editor#108]
Hi,
I think your library is really great.
As an improvement, it would be great if the whole form could be easely "themed" using CSS.
Currently, almost everything is "inline styled". Of course, it is still possible to make a custom theme and overide each method to return a new element with solely a css class for it. The small issue is that style elements are sometimes overriden outside of theme utility classes. It'd be nice to have a default theme without any inline style and a sample css out of the box.
Cheers,
Arnaud

**Comment by MaKleSoft** [comment]
I completely agree with @Reggino. Imo, `AbstractTheme` has no business adding any inline styles at all except styles that are strictly required for functional purposes (like `display`, `position` and `z-index`). The [getModal](https://github.com/jdorn/json-editor/blob/master/src/theme.js#L22-L31) method is a good example for unnecessary styling. Styles like `border` and `box-shadow` should be set via css classes, **not** via inline styling. Currently all themes that want to change this need to explicitly override the inline styling or use the `!important` directive in their css classes which clearly is less than optimal. Please consider removing all unnecessary styling from `AbstractTheme`. @jdorn
